installer/project/components/configure_war.xml

```diff
@@ -284,9 +284,8 @@
         </propertiesFileSet>
         <propertiesFileSet>
           <file>${temp_dir}/conf/common/security.properties</file>
-          <key>security.help.about</key>
```

Comment:
This removal seems dangerous for a point release. I realize it's set to an empty string in the various build processes we now use, but total removal of the key might have downstream effects that we aren't aware of.

Reply:
I'm worried about leaving it because I've seen prop files with a newline in the value. This prop key isn't used by any part of the code.